### PR TITLE
Added additional details to SelectMultiple `onMore` and `onSearch`

### PR DESCRIPTION
**Pull request by jcfilben**

### CODE REVIEW

**Review by britt6612 — APPROVED** [review]

File: src/screens/SelectMultiple.js [hunk 1]
```diff
@@ -305,7 +305,8 @@ const SelectMultiplePage = () => (
           <Description>
             Function that will be called when the user types in the search
             input. If this property is not provided, no search field will be
-            rendered.
+            rendered. New options can be added to 'options' based on a search
```
Comment (taysea) [inline src/screens/SelectMultiple.js:308]: I'm wondering if this sentence would make sense to someone less familiar with the pattern. is there anything that could be added in the example code snippet that might make this more clear?

Comment (taysea) [reply src/screens/SelectMultiple.js:308]: I think that wording is great. Maybe just "search term" as opposed to "search term query".

Comment (jcfilben) [reply src/screens/SelectMultiple.js:308]: Sounds good, I'll update and push up changes

**Review by taysea — APPROVED** [review]
LGTM.

**Pull request merged**